content/pages/getting_started/what_do_i_need.md

```diff
@@ -0,0 +1,96 @@
+
+* Eye protection - ANSI-rated [face shield][] or safety glasses
+* Hearing protection - [earplugs][] or muffs
+* Lung protection (for grinding and handling particulates) - P100 or better [respirator][]
+
+## The Basics: Everything you need to get started
+
+The core requirements for a blacksmith's shop are a hammer, an anvil, a forge, and probably some tongs. While it is possible to make tongs without having tongs, there's no shame in buying your first (or first several) set of tongs. If you're on a tight budget, [Ken's Custom Iron tong blanks][] are a great middle-ground option.
+
+These are our core equipment recommendations for every budget.
+
+### Shoestring
+
+Suggestions for those looking to get started as cheaply as possible.
+
+* Hammer: 2-2.5lb (900-1100g) crosspein, rounding, or drilling hammer from your local big box store. Check out our guide to [dressing your hammer][].
+* Forge: [Just a Box of Dirt][jabod]
+* Anvil: [Sledgehammer head][sledgehammer anvil] or 4140/1045 offcut from eBay or your local steelyard or machine shop
+* Tongs: Use long stock or [Ken's Custom Iron tong blanks][].
+
+### Economy
+
+If you've got a little more cash to burn, these options might make your life a little easier.
+
+* Hammer: 2-2.5lb (900-1100g) crosspein, rounding, or drilling hammer from your local big box store. Check out our guide to [dressing your hammer][].
+* Forge: [Mr. Volcano single burner][Mr. Volcano forge]
+* Anvil: [Harbor Freight Doyle cast steel anvil][] or check out our [anvil buying guide][]
+* Tongs: 3/8" V-bit bolt jaw or wolf jaw from [Centaur Forge][], [Blacksmiths Depot][], or [Texas Farrier Supply][].
+
+### Premium
+
+Fine goods for the discerning aspiring blacksmith
+
+* Hammer: 2-2.5lb (900-1100g) crosspein or rounding hammer from one of your fellow artists: [Brent Bailey][]
+* Forge: [Mr. Volcano single or double burner][Mr. Volcano forge] or [Chile forge][]
+* Anvil: [Anvil buying guide][] or buy new from [Texas Farrier Supply][]
+* Tongs: 3/8" V-bit bolt jaw or wolf jaw from [Centaur Forge][], [Blacksmiths Depot][], or [Texas Farrier Supply][]
+
+## Nice-to-Haves: Upgrading your shop
+Forging with just the basics will get frustrating fast. A post vise is an excellent upgrade - unlike conventional bench vises, a post vise is designed to stand up to repeated heavy hammering by transferring the force of the hammer blows through the post into the floor.
+
+For folks interested in bladesmithing, a belt grinder is ideal; the bulk of bladesmithing operations take place at the grinder. If you're on a budget, a [1x30][] belt grinder will get you started, but if you have a bit more to spend, a [2x72][] is a worthwhile investment. However, if a belt grinder isn't an option for you, an angle grinder or even a set of [files][Pferd files], together with a bit of patience, will yield perfectly respectable results.
```

Comment:
Based on recent discussion in Hammerbros, I think we should keep everything here, but maybe separate them out (so it doesn't look like grinders and files are equivalent); so my suggestion is something like
```md
#### Grinder
* Useful for removing stock quickly, beveling, and sharpening
* Belt grinders are nice, but 2x72s are expensive; if out of your budget, consider a 1x30 or angle grinder
* Most smiths will eventually have multiple different grinders for different tasks, so don't overthink your choice
* Uses disposable belts and discs with different grits/purposes
#### Files
* Can also be used for stock removal (albeit slower)
* Can be used in situations where grinders can't (difficult angles, finer control, etc)
* Different types of files have different sizes/teeth; consider a file set
```

Just my $0.02 though; it seems like everyone has a different view on the whole "what is most important to start bladesmithing" topic so... eh, I guess?